### DISCUSSION TITLE
Is this repository maintained?

### DISCUSSION THREAD

**Discussion by temeddix** [domlysz/BlenderGIS#744] [General]
I wanted to make a contribution and make a pull request(related to SHP file encoding problem), but it seems that this project is stopped more than half a year. There are many pull requests waiting for admission for years, so...it's quite discouraging to invest my time into it. If the code is quality enough, is there any possibility that it will be merged into main branch and get released?

**Comment by atseewal** [comment]
I would say it's not actively being developed. The only update in years was because a major functionality needed updating (SRTM data started requiring an API key).
